packages/account/src/hdwallet/hdwallet.ts

```diff
@@ -221,7 +221,8 @@ class HDWallet {
   }
 
   static fromExtendedKey(extendedKey: string) {
-    const decoded = toBeHex(decodeBase58(extendedKey));
+    let decoded = toHex(decodeBase58(extendedKey));
+    decoded = `${decoded.substring(0, 2)}0${decoded.substring(2)}`;
```

Comment:
This was a strange addition on my part needed to make the tests pass, essentially `toHex` omits a padded `0` at the third index, thus `0x0488b21e03587302...` becomes `0x488b21e03587302....` but I would think that  if `toHex` had a bug it would've been discovered much earlier... any thoughts?

Reply:
This is weird but I agree. `toHex` also just wraps `BN` functionality. Should we not be updating the test case rather than having this specific logic in place?

Reply:
@danielbate The [test cases look right](https://github.com/FuelLabs/fuels-ts/blob/master/packages/account/src/hdwallet/hdwallet.test.ts#L88-L129) as they are just key derivation